api/controllers/getAnimeByIdController.go

```diff
@@ -0,0 +1,55 @@
+package controllers
+
+import (
+	animeU "bootCampApi/api/usecases"
+	"encoding/csv"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/fatih/structs"
+	"github.com/gorilla/mux"
+)
+
+func GetAnimeByIdC(w http.ResponseWriter, r *http.Request) {
+	id := mux.Vars(r)["id"]
+
+	animeData := animeU.GetAnimeById(id)
+	animeValues := make([]string, 0)
+	for _, v := range structs.Values(animeData) {
+		temp := fmt.Sprint(v)
+		animeValues = append(animeValues, temp)
```

Comment:
There's no need to create a new variable.
```suggestion
		animeValues = append(animeValues, fmt.Sprint(v))
```